src/main/scala/vectorpipe/osm/package.scala

```diff
@@ -176,7 +177,8 @@ package object osm {
    *     across its child members. Otherwise, Relations are "dropped"
    *     from the output.
    */
-  def toFeatures(nodes: RDD[Node], ways: RDD[Way], relations: RDD[Relation]): RDD[OSMFeature] = {
+  def toFeatures(nodes: RDD[Node], ways: RDD[Way], relations: RDD[Relation])
+                (logError: (Feature[Line, ElementData] => String) => Feature[Line, ElementData] => Unit): RDD[OSMFeature] = {
```

Comment:
Consider the same point about order of argument lists from above

Reply:
Since there's only one tacked-on function here, I'd lean on Scala convention. Someone might want to:
```scala
toFeatures(nodes, ways, rels) { f => { a => ??? } }
```